tests/test_costs.py

```diff
@@ -1,18 +1,11 @@
 import pytest
-from ruptures.costs import (
-    CostAR,
-    CostCLinear,
-    CostL1,
-    CostL2,
-    CostLinear,
-    CostNormal,
-    CostRank,
-    CostRbf,
-    cost_factory,
-)
+import numpy as np
+from ruptures.costs import cost_factory
 from ruptures.datasets import pw_constant
 from ruptures.exceptions import NotEnoughPoints
 
+np.random.seed(101)
```

Comment:
This affects the global state of random, use RandomState instead (see [this SO answer](https://stackoverflow.com/questions/5836335/consistently-create-same-random-numpy-array/5837352#5837352) from a numpy dev)

Reply:
Will have a look, thx ! 
See the comment below since once we address the (possible) issue, then we might be able to remove the random state enforcement.

Reply:
Done !